utils/helm/speckle-server/templates/_helpers.tpl

```diff
@@ -569,7 +569,7 @@ Generate the environment variables for Speckle server and Speckle objects deploy
 - name: FF_WORKSPACES_SSO_ENABLED
   value: {{ .Values.featureFlags.workspacesSSOEnabled | quote }}
 
-- name: FF_WORKSPACES_NEW_PLAN_ENABLED
+- name: FF_WORKSPACES_NEW_PLANS_ENABLED
```

Comment:
@alemagio I think u may have created a duplicate feature flag, Gergo already created the new plans FF in his initial PR

@iainsproat I left the helm chart path as is, maybe thats fine if these are supposed to be somewhat temporary